syntax = 'proto3';

package proto;

option go_package = "github.com/aaarrti/2048-grpc/proto";
import "google/protobuf/empty.proto";


message IntMessage {
  int32 value = 1;
}


service EnvService {

  rpc provisionEnvironment (google.protobuf.Empty) returns (IntMessage);
  rpc stopEnvironment (IntMessage) returns (google.protobuf.Empty);

}